docs/en-US/component/scrollbar.md

```diff
@@ -39,28 +39,37 @@ scrollbar/manual-scroll
 
 :::
 
+## Infinite scroll ^(2.9.11)
+
+:::demo When the end of the scroll is reached `endReached` is called. It can be used for infinite scroll.
```

Comment:
If you want to support four directions in the future, then it's best to call it `bottom-reached` or something like that.